vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/BiodataVariantToVariantContextConverterTest.java

```diff
@@ -400,6 +402,43 @@ public void twoStudiesConflictingNamesSingleVariant() {
         checkVariantContext(variantContext, CHR_1, 1000, 1000, "T", "G", variant.getSourceEntries(), true);
     }
 
+
+    @Test
+    public void csqAnnotation() {
+        // create variant
+        VariantSource variantSource = createTestVariantSource(STUDY_1);
+        String variantLine = String
+                .join("\t", CHR_1, "1000", "id", "C", "A", "100", "PASS", ".", "GT", "0|0", "0|0", "0|1", "1|1", "1|1",
+                      "0|1");
+        List<Variant> variants = variantFactory.create(variantSource, variantLine);
+        assertEquals(1, variants.size());
+        // populate variant with test csq data
+        variants.get(0).getAnnotation().setAlternativeAllele("aAllele");
+        List<ConsequenceType> consequenceTypes = new ArrayList<>();
+        List<String> soNames = new ArrayList<>(Arrays.asList("regulatory_region_ablation", "3_prime_UTR_variant"));
+        ConsequenceType consequenceType = new ConsequenceType("gene", "ensembleGeneId",
+                                                              "EnsembleTransId", "strand",
+                                                              "bioType", 10, 10, 10,
+                                                              "aaChange", "codon", new ArrayList<>(), soNames);
+        ConsequenceType consequenceType2 = new ConsequenceType("gene2", "ensembleGeneId2",
+                                                              "EnsembleTransId2", "strand2",
+                                                              "bioType2", 20, 20, 20,
+                                                              "aaChange2", "codon2", new ArrayList<>(), soNames);
+        consequenceTypes.add(consequenceType);
+        consequenceTypes.add(consequenceType2);
+        variants.get(0).getAnnotation().setConsequenceTypes(consequenceTypes);
+        // export variant
+        BiodataVariantToVariantContextConverter variantConverter =
+                new BiodataVariantToVariantContextConverter(Collections.singletonList(variantSource),
+                                                            noSampleNamesConflictSampleNameCorrections);
+        VariantContext variantContext = variantConverter.transform(variants.get(0));
+        // test if CSQ is properly transformed
+        assertTrue(!variantContext.getCommonInfo().getAttributes().isEmpty());
+        String csq = (String) variantContext.getCommonInfo().getAttribute("CSQ");
+        assertNotNull(csq);
+        assertEquals("aAllele|regulatory_region_ablation&3_prime_UTR_variant|gene|ensembleGeneId|EnsembleTransId|bioType|10|10,aAllele|regulatory_region_ablation&3_prime_UTR_variant|gene2|ensembleGeneId2|EnsembleTransId2|bioType2|20|20", csq);
+    }
+
```

Comment:
Can you create a new test (or add to some of the existing ones) a check that, if no consequence types are added to the variant, then the CSQ info tag is not created?